impl/src/fmt/mod.rs

```diff
@@ -509,16 +570,47 @@ impl Placeholder {
 /// are allowed.
 ///
 /// [`fmt::Display`]: std::fmt::Display
-#[derive(Debug, Default)]
-struct ContainerAttributes {
+#[derive(Debug)]
+struct ContainerAttributes<T> {
     /// Interpolation [`FmtAttribute`].
     fmt: Option<FmtAttribute>,
 
     /// Addition trait bounds.
     bounds: BoundsAttribute,
+
+    /// Rename unit enum variants following a similar behavior as [`serde`](https://serde.rs/container-attrs.html#rename_all).
+    rename_all: Option<T>,
```

Comment:
@ModProg instead of modifying the `ContainerAttributes` with an additional slot (which is useless for the `Debug`), I think it's better to leave it "as is", and:
1. Move `RenameAllAttribute` into `display` module.
2. Inside `display` module do:
    ```rust
    struct ContainerAttributes {
        rename_all: RenameAllAttribute,
        common: super::ContainerAttributes,
    }
    ```
    and parse it as `Either<RenameAllAttribute, super::ContainerAttributes>` to omit repeating parsing of the `super::ContainerAttributes` twice.

Reply:
I thought It'd make sense to allow to have `rename_all` and `bounds` in the same attribute. But that will only at most save one line anyway.